skore-ui/src/components/PlotlyWidget.vue

```diff
@@ -28,7 +29,7 @@ const resizeObserver = new ResizeObserver(() => {
 onMounted(async () => {
   if (container.value) {
     resizeObserver.observe(container.value);
-    const plot = await newPlot(container.value, props.spec.data, makeLayout());
+    const plot = await react(container.value, props.spec.data, makeLayout());
     if (props.spec.frames) {
       addFrames(plot, props.spec.frames);
```

Comment:
Should this snippet be factorize in a `buildPlot` function and used here and in the watch ?